apollo-runtime/src/main/java/com/apollographql/android/impl/RealApolloCall.java

```diff
@@ -162,4 +187,29 @@ static String cacheKey(RequestBody requestBody) throws IOException {
     requestBody.writeTo(hashBuffer);
     return hashBuffer.readByteString().md5().hex();
   }
+
+  private static String cacheControlHeader(CachePolicy cachePolicy) {
```

Comment:
I feel like this can be rolled up into the enum as string values. So we don't need this mapping and can just do cachePolicy.toString() or .headerValue() smth like http://stackoverflow.com/questions/3978654/best-way-to-create-enum-of-strings